docker-compose.yml

```diff
@@ -2,13 +2,13 @@ version: "2.2"
 
 services:
   dash_server:
-    build: graphs/
+    build: covid_graphs/
     image: accu_dash
     volumes:
-      - "./graphs:/covid19/graphs:ro"
+      - "./covid_graphs:/covid19/covid_graphs:ro"
       - "./data:/covid19/data:ro"
     working_dir: "/covid19"
     ports:
       - 127.0.0.1:8081:8080
     restart: always
-    command: "/covid19/graphs/server.py data/ data/results-poly.pb data/results-exp.pb"
+    command: "/covid19/covid_graphs/covid_graphs/scripts/server.py data/ data/results-poly.pb data/results-exp.pb"
```

Comment:
```suggestion
    command: "server.py data/ data/results-poly.pb data/results-exp.pb"
```

Reply:
I was not able to make `index.html` load by running the global version of `server.py` script. I suggest to keep for now.